lib/Catmandu/Store/Datahub/API.pm

```diff
@@ -90,7 +91,8 @@ sub add {
     my $token = $self->access_token;
     my $response;
 
-    $response = $self->client->post($url, Content_Type => 'application/lido+xml', Authorization => sprintf('Bearer %s', $token), Content => $data);
+    my $enc_data = encode_utf8($data);
```

Comment:
Why do you this here? What's the added benefit of doing an encode_utf8()?

Reply:
If `$data` contains non-ascii-characters, the client will fail.